src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/sdk/devtestlabs/__init__.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
```

Comment:
I know but it's being already reviewed at https://github.com/vishrutshah/azure-cli/pull/7